solutions/usaco-892.mdx

```diff
@@ -74,4 +74,35 @@ public class SleepyCowSorting {
 	
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int arr[105];
+
+int main() {
+    	freopen("sleepy.in", "r", stdin);
+    	freopen("sleepy.out", "w", stdout);
+	int n; cin >> n;
+	for (int i = 0; i < n; i++) {
+		cin >> arr[i];
+	}
+	int answer = n - 1;
+	for (int i = n - 2; i >= 0; i--) {
+		if(arr[i] < arr[i+1]) {
```

Comment:
other people don't want this if else style but I'm fine with it

Reply:
uh what do you mean